Instructions.txt

```diff
@@ -0,0 +1,132 @@
+########################################
+### Prepare system (Linux Mint 18.1) ###
+########################################
+
+### Step 1: Create Project Directory
+mkdir TKproj
+cd TKproj
+
+
+### Step 2: Clone sources from github
+git clone https://github.com/kevinhughes27/TensorKart.git
+git clone https://github.com/kevinhughes27/mupen64plus-input-bot.git
+git clone https://github.com/mupen64plus/mupen64plus-core.git
+git clone https://github.com/mupen64plus/mupen64plus-input-sdl.git
+
+
+### Step 3: Install dependancies
+sudo apt-get install dpkg-dev build-essential swig python2.7-dev libwebkitgtk-dev libjpeg-dev libtiff-dev checkinstall ubuntu-restricted-extras freeglut3 freeglut3-dev libgtk2.0-dev  libsdl1.2-dev libgstreamer-plugins-base0.10-dev 
```

Comment:
I didn't realise this much `apt-get`ing was required. Are `freeglut3`, `swig`, `ubuntu-restricted-extras` needed? they look suspect to me. I previously built opencv on my system so maybe that process made this project easier to setup for me. Another option we could explore is to build a docker container for TensorKart and then all this documentation becomes setup code that we can use instead of documentation we have to update.

also there is an extra space after libgtk2.0-dev